libbeat/common/transport/tlscommon/server_config.go

```diff
@@ -31,8 +31,8 @@ type ServerConfig struct {
 	VerificationMode TLSVerificationMode `config:"verification_mode"` // one of 'none', 'full'
 	Versions         []TLSVersion        `config:"supported_protocols"`
 	CipherSuites     []tlsCipherSuite    `config:"cipher_suites"`
-	CAs              []string            `config:"certificate_authorities"`
-	Certificate      CertificateConfig   `config:",inline"`
+	CAs              []string            `config:"certificate_authorities" validate:"required"`
```

Comment:
I was thinking this would be conditionally required. If `client_auth` is `none` then you wouldn't need to set this.

Reply:
Oh the `none` case :(